topology/enhancers/topology_manager.go

```diff
@@ -227,27 +231,28 @@ func (tm *TopologyManager) handleNodeRuleRequest(action string, resource types.R
 	return nil
 }
 
-func (tm *TopologyManager) handleEdgeRuleRequest(action string, resource types.Resource) {
+func (tm *TopologyManager) handleEdgeRuleRequest(action string, resource types.Resource) error {
 	edge := resource.(*types.EdgeRule)
 	switch action {
 	case "create", "set":
-		tm.createEdge(edge)
+		return tm.createEdge(edge)
 	case "delete":
 		src := tm.getNodes(edge.Src)
 		dst := tm.getNodes(edge.Dst)
 		if len(src) < 1 || len(dst) < 1 {
 			logging.GetLogger().Errorf("Source or Destination node not found")
-			return
+			return nil
 		}
 
 		for {
 			if link := tm.graph.GetFirstLink(src[0], dst[0], edge.Metadata); link != nil {
 				tm.graph.DelEdge(link)
```

Comment:
DelEdge can now return an error so we may at least log it

Reply:
done